aws-rollback.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import sys
+
+TARGET_TAG = 'latest'
+ECS_CLUSTER = 'rust-ecs-prod'
+
+def main():
+    """CLI entrypoint of the program."""
+    if len(sys.argv) < 2:
+        usage()
+    repository_name = sys.argv[1]
+    images = get_images(repository_name)
+    while True:
+        selected_image_index = let_user_pick_image(images)
+        if selected_image_index == -1:
+            exit(0)
+
+        if selected_image_index is not None:
+            break
+
+    image = images[selected_image_index]
+    if image["imageTag"] == TARGET_TAG:
+        err(f"selected image already tagged as {TARGET_TAG}")
+
+    eprint(f"selected option: {image}")
+    manifest = get_image_manifest(repository_name, image["imageTag"])
+    retag_image(repository_name,manifest)
+    eprint(f"image {image} retaged as '{TARGET_TAG}'")
+    if can_redeploy(repository_name):
+        redeployed = force_redeploy()
+        if redeployed:
+            eprint(f"successfully rollback and re-deploy")
```

Comment:
It'd be nice here to show success message even for other cases:

* Successfully rolled back the image, but no service with the same name to redeploy found
* Successfully rolled back the image, but redeploying the service with the same name failed

Reply:
Hi @pietroalbini, thanks for the feedback. I just pushed all the changes.
Thx!